config_example.sh

```diff
@@ -19,6 +19,9 @@ set -x
 # NOTE: dual stack is not expected to fully work yet.
 #export IP_STACK=v4
 
+# BMC type. Valid values are redfish, redfish-virtualmedia, or ipmi.
+#export BMC_DRIVER=redfish-virtualmedia
```

Comment:
For other reviewers, this is consumed in the metal3-dev-env ansible roles:

https://github.com/metal3-io/metal3-dev-env/blob/master/vm-setup/roles/libvirt/tasks/vm_setup_tasks.yml#L129